projects/inchi/inchi_input_fuzzer.cc

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Google Inc.
```

Comment:
A quick file-level comment: `std::numeric_limits`, `extern "C"`, and `const_cast` are all C++ features not found in C.  If you remove these, I think you should be able to build the fuzzer as a pure-C fuzzer.